Helm/opensearch/templates/ingress.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "opensearch.uname" . -}}
+{{- $servicePort := .Values.httpPort -}}
+{{- $ingressPath := .Values.ingress.path -}}
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
This should use `networking.k8s.io/v1` for kubernetes v1.19.0+, otherwise we get deployment warnings and failures from v1.22.0+

https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122

Reply:
Thanks @viceice for pointing out, I will upgrade the versions.